packages/core/src/base-chart.ts

```diff
@@ -73,6 +74,33 @@ export class BaseChart {
 		}
 	}
 
+	styleHolderElement() {
+		const holderElement = this.holder as HTMLElement;
+		const { width, height } = this.options;
+
+		// If width exists in options
+		if (width) {
+			// Validate it's value
+			if (Tools.validateWidthHeightValues(width)) {
+				// Apply formatted width attribute to chart
+				holderElement.style.width = Tools.formatWidthHeightValues(width);
```

Comment:
I see you're trying to enforce using sizes in % or px but I'm having a hard time understanding why are those only viable options.